index.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Country Documentation</title>
+  <link rel="stylesheet" href="style.css">
+</head>
```

Comment:
> * [ ] Kindly Use `<meta http-equiv="X-UA-Compatible" content="IE=edge">`
                        `<meta name="viewport" content="width=device-width, initial-scale=1.0">` These tags before line `6` , which would improve your site's accessibility rating, SEO rating, and user experience.  😃